app/core/api/thang-types.coffee

```diff
@@ -0,0 +1,14 @@
+fetchJson = require './fetch-json'
+utils = require 'core/utils'
+
+module.exports = {
+  getHeroes: (options) ->
+    data = {
+      view: 'heroes'
+    }
+    if options?.project
+      _.assign data, {
+        project: options.project.join(',')
+      }
+    return fetchJson('/db/thang.type', { data })
```

Comment:
All options need to be passed on to fetchJson, for example if `headers` is included, or any other `fetch` options property. So

```
options = _.cloneDeep(options)
options.data ?= {}
options.data.view = 'heroes'
if options.project
  _.assign options.data, {
    project: options.project.join(',')
  }
  return fetchJson('/db/thang.type', options)